### PR TITLE
Update EventCallbacks

### PR DESCRIPTION
**Pull request by luanluciano93**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

**Protocol version**
8.60

### Changes Proposed
Updates EventCallbacks as per the official TFS github repo.

**Issues addressed:** 
I was having problems executing more than one onMoveItem function.

### CODE REVIEW

File: data/events/scripts/player.lua [hunk 1]
```diff
@@ -145,29 +145,29 @@ function Player:onGainExperience(source, exp, rawExp)
 		useStamina(self)
 
 		local staminaMinutes = self:getStamina()
-		if staminaMinutes > 2400 and self:isPremium() then
+		if staminaMinutes > 2340 and self:isPremium() then
```
Comment (EPuncker) [inline data/events/scripts/player.lua:148]:
```suggestion
		if staminaMinutes > 2400 and self:isPremium() then
```

**Review by EPuncker — COMMENTED** [review]
the `onMoveItem` changes won't work unless the necessary source changes gets made as well

**Comment by luanluciano93** [comment]
@nekiro @EPuncker ?

**Comment by luanluciano93** [comment]
@nekiro ?

**Pull request closed without merging**